google-cloud-firestore/src/main/java/com/google/cloud/firestore/Firestore.java

```diff
@@ -168,9 +169,29 @@ void getAll(
   @Nonnull
   WriteBatch batch();
 
+  /**
+   * Creates a {@link BulkWriter} instance, used for performing multiple writes in parallel.
+   * Gradually ramps up writes as specified by the 500/50/5 rule.
+   *
+   * @see <a href=https://cloud.google.com/datastore/docs/best-practices#ramping_up_traffic>Ramping
+   *     up traffic</a>
+   */
+  @BetaApi
   @Nonnull
   BulkWriter bulkWriter();
 
+  /**
+   * Creates a {@link BulkWriter} instance, used for performing multiple writes in parallel.
+   * Gradually ramps up writes as specified by the 500/50/5 rule.
```

Comment:
This isn't necessarily true for all `options` object, is it?

Reply:
Added "unless otherwise configured by a BulkWriterOptions object".